[2024-05-03T22:55:06.931+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-03T22:55:02.589600+00:00 [queued]>
[2024-05-03T22:55:06.945+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-03T22:55:02.589600+00:00 [queued]>
[2024-05-03T22:55:06.946+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-03T22:55:06.947+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2024-05-03T22:55:06.947+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-03T22:55:06.958+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): transform_data> on 2024-05-03 22:55:02.589600+00:00
[2024-05-03T22:55:06.965+0000] {standard_task_runner.py:55} INFO - Started process 2359 to run task
[2024-05-03T22:55:06.970+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'covid_analytics_dag', 'transform_data', 'manual__2024-05-03T22:55:02.589600+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/covid_analytics.py', '--cfg-path', '/tmp/tmpqaigr3bw']
[2024-05-03T22:55:06.971+0000] {standard_task_runner.py:83} INFO - Job 81: Subtask transform_data
[2024-05-03T22:55:07.028+0000] {task_command.py:376} INFO - Running <TaskInstance: covid_analytics_dag.transform_data manual__2024-05-03T22:55:02.589600+00:00 [running]> on host 2cb8e808f114
[2024-05-03T22:55:07.081+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Diego Munera
AIRFLOW_CTX_DAG_ID=covid_analytics_dag
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2024-05-03T22:55:02.589600+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-03T22:55:02.589600+00:00
[2024-05-03T22:55:07.098+0000] {credentials.py:1180} INFO - Found credentials in environment variables.
[2024-05-03T22:55:12.012+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-05-03T22:55:12.071+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=covid_analytics_dag, task_id=transform_data, execution_date=20240503T225502, start_date=20240503T225506, end_date=20240503T225512
[2024-05-03T22:55:12.138+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-03T22:55:12.157+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
